requirements.txt

```diff
@@ -0,0 +1,15 @@
+arrow==0.14.0
```

Comment:
do we really need all these packages???

Reply:
Not really, I am just a novice to Python and I was trying to mimic other repositories online. So I have ended up with this.

So now I have made changes to the requirements.txt and added some lines in the gitignore file. (gitignore file is built from gitignore.io).